### DISCUSSION TITLE
How to downloading the lastest documentiation?

### DISCUSSION THREAD

**Discussion by cmuMenglab** [rdkit/rdkit#4402] [Q&A] [answered]
Hi,postdoctoral fellow:
Recently I want download the PDF version about RDKit 2021.03.1 documentation,and how do I get this?
I'd be grateful if you reply me qiuckly !

**Comment by greglandrum** [comment, marked as answer]
We don't create a PDF version of the documentation. You can dowload the HTML version at rdkit.org.
https://rdkit.org/RDKit_Docs.current.tgz

**Reply by cmuMenglab** [reply]
Thanks, @greglandrum, much appreciated
